### ISSUE TITLE
Enforce canonical classes suggests classes that breaks calc by removing whitespace

### ISSUE THREAD

**Issue by TeamAcarup** [schoero/eslint-plugin-readable-tailwind#375]
### Description

Just noticed an issue in one of my project.

I have the following tailwind class defined on one of my components:

```
lg:left-[calc(-1*(var(--desktop-logo-width)+var(--header-padding)+var(--header-gap)))]
```

Which translates to this property in CSS:

```
calc(-1 * (var(--desktop-logo-width) + var(--header-padding) + var(--header-gap)))
```

The `enforce-canonical-classes` plugin suggests using the following short-hand:
```
lg:-left-[(var(--desktop-logo-width)+var(--header-padding)+var(--header-gap))]
```

Which gets translated to the following CSS property:

```
calc((var(--desktop-logo-width)+var(--header-padding)+var(--header-gap)) * -1)
```

While it looks similar, the missing white space around the "+" break the calculation.

### Flavor

TSX

### Code Input

```tsx
lg:-left-[(var(--desktop-logo-width)+var(--header-padding)+var(--header-gap))]
```

### Expected Behavior

```tsx
calc(-1 * (var(--desktop-logo-width) + var(--header-padding) + var(--header-gap)))
```

### Actual Behavior

```tsx
calc((var(--desktop-logo-width)+var(--header-padding)+var(--header-gap)) * -1)
/* No whitespace around the "+", breaks the calc */
```

### ESLint Log

```shell
No output
```

### ESLint Config

```typescript
eslintPluginBetterTailwindcss.configs.recommended,
  {
    files: ['**/*.{js,jsx,ts,tsx}'],
    settings: {
      'better-tailwindcss': {
        entryPoint: 'app/tailwind.css',
        tailwindConfig: 'tailwind.config.ts',
      },
    },
    // if needed, override rules to configure them individually
    rules: {
      'better-tailwindcss/no-unknown-classes': 'off',
      'better-tailwindcss/enforce-consistent-line-wrapping': [
        'warn',
        {printWidth: 100, lineBreakStyle: 'unix'},
      ],
    },
  },
```

### Versions

- ESLint: 9.39.4
- Parser: 8.59.1 ? (Not sure about this one)
- Plugin: 4.5.0
- Node: 22.17.0

**Comment by schoero** [comment]
The logic for the `enforce-canonical-classes` rule lives entirely inside Tailwind CSS. The plugin simply calls Tailwind’s internal functions and does not influence how classes are canonicalized.

If you’re running into an issue, it would be best to report it in the Tailwind CSS repository: https://github.com/tailwindlabs/tailwindcss/issues

**Comment by TeamAcarup** [comment]
Alright, thank you. I'll reopen an issue upstream.
